sessions/react-state/demo-food-order-start/README.md

```diff
@@ -0,0 +1,47 @@
+# React State: Demo Food Order Start
```

Comment:
Should we empty this readme or write a single sentence to inform that this is a demo?